site/src/docs/components/koros/accessibility.mdx

```diff
@@ -10,5 +10,7 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 ## Accessibility
 
 ### Pay attention to
-- HDS Footer is designed to support many different brand colours. When customising colours, refer to <InternalLink href="/foundation/design-tokens/colour">colour guidelines</InternalLink> to ensure accessibility.
-- **When designing the footer link hierarchy, always think about keyboard and screen reader users.** Remember that keyboard users need to manually navigate to each element.
+
```

Comment:
For accessibility I would suggest:
Koros are decorative items and should be used in background and hidden from screen readers. See e.g. [Hero](https://hds.hel.fi/components/hero) and [Footer](https://hds.hel.fi/components/footer).
Note that in this documentation the koros have alt texts as they present the component and are therefore important content.

Koros are not an interactive element and do not receive focus.